source/adminguide/host_and_storage_tags.rst

```diff
@@ -96,23 +105,31 @@ Storage tags are responsible for directing volumes to compatible primary storage
 To explain the behavior of storage tags, some examples will be demonstrated:
 
 #. Tag setup:
+
     * Storage: A
     * Offering: A,B
+
     Storage and offering accept a list of tags, with the comma (,) being their separator. Therefore, in this example, the offering has tags A and B. In this example, it will not be possible to allocate the volume, as all the offering tags must exist in the storage. Although the storage has the A tag, it does not have the B tag.
 
 #. Tag setup:
+
     * Storage: A,B,C,D,X
     * Offering: A,B,C
+
     In this example, it will be possible to allocate the volume, as all the offering tags exist in the storage.
 
 #. Tag setup:
+
     * Storage: A, B, C
     * Offering: (no tag)
+
     In this example, it will be possible to allocate the volume, as the offering does not have any tag requirements.
 
 #. Tag setup:
+
     * Storage: (no tag)
     * Offering: D,E
+
     In this example, it will not be possible to allocate the volume, as the storage does not have tags, therefore it does not meet the offering requirements.
```

Comment:
```suggestion
In this example, it will not be possible to allocate the volume, as the storage does not have tags, therefore it does not meet the offering requirements.
```

Reply:
As above, it might be better to keep the indentation.